org.eclipse.xtext.ide.tests/testlang-src-gen/org/eclipse/xtext/ide/tests/testlanguage/AbstractPartialContentAssistTestLanguageRuntimeModule.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016 TypeFox GmbH (http://www.typefox.io) and others.
+ * Copyright (c) 2016, 2017 TypeFox GmbH (http://www.typefox.io) and others.
```

Comment:
2017?

Reply:
i just wanted to fix the bad rework from before so that the generated stuff is consistent.